smaug/operators/batch_norm_op.h

```diff
@@ -115,7 +118,9 @@ class BatchNormOp : public FusedActivationOp {
     SamplingInfo sampling;
 };
 
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
```

Comment:
This looks awkward. What's the reason that Doxygen doesn't like this?

Reply:
REGISTER_SPECIAL_OP looks like a function-call, so by default, it'll get treated like a function and show up as `smaug::REGISTER_CREATE_OP(Operator, Backend)`. But it actually expands to just a template specialization. And if I force expansion, Breathe picks it up as a variable, strangely, not even a function, then throws a ton of warnings.

This is an issue with Breathe, not with Doxygen, but the only way to address it is at its source. With regular Doxygen, this doesn't even show up at all.

Reply:
This is just one of the many other reasons I'm increasingly inclined to just use Doxygen instead of Sphinx. Sphinx is good for Python, but there are so many issues with the way it handles C++ APIs, and while I could probably address them given more time, I just don't want to spend that time on documentation customizing, when there's so many more important things.